ili/inference/runner_sbi.py

```diff
@@ -99,48 +109,76 @@ def load_object(cls, config: Dict) -> Any:
         )(**config["args"])
 
     @classmethod
-    def load_inference_model(
+    def load_inference_class(
+        cls,
+        inference_module: str,
+        inference_class: str,
+    ) -> NeuralInference:
+        """Neural inference class used to train posterior
+
+        Args:
+            inference_module (str): module from which to import class
+            inference_class (str): class name
+
+        Returns:
+            NeuralInference: neural inference class
+        """
+        module = importlib.import_module(inference_module)
+        return getattr(module, inference_class)
```

Comment:
In this, we can't specify additional arguments of the inference class. e.g. logging_level